apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1.IntegrationTests/DatastoreFixture.cs

```diff
@@ -31,12 +31,30 @@ public sealed class DatastoreFixture : CloudProjectFixtureBase, ICollectionFixtu
         private const int RetryCount = 10;
         private static readonly TimeSpan RetryDelay = TimeSpan.FromSeconds(3);
 
+        public DatastoreDb DatastoreTestDb { get; }
+
         public string NamespaceId { get; }
         public PartitionId PartitionId => new PartitionId { ProjectId = ProjectId, NamespaceId = NamespaceId };
 
         public DatastoreFixture()
         {
             NamespaceId = IdGenerator.FromDateTime(prefix: "test-");
+            DatastoreTestDb = CreateTestStudents();
```

Comment:
I would remove all the changes from DatastoreFixture - with the above suggestion with only one test, we can populate the data in the test itself, as we do for the other integration tests.